src/lib/isLicensePlate.js

```diff
@@ -6,32 +6,20 @@ const validators = {
   'de-LI': str => /^FL[- ]?\d{1,5}[UZ]?$/.test(str),
+  'en-IN': str => /^[A-Z]{2}[ -]?[0-9]{1,2}(?:[ -]?[A-Z])(?:[ -]?[A-Z]*)?[ -]?[0-9]{4}$/.test(str),
+  'es-AR': str => /^(([A-Z]{2} ?[0-9]{3} ?[A-Z]{2})|([A-Z]{3} ?[0-9]{3}))$/.test(str),
   'fi-FI': str => /^(?=.{4,7})(([A-Z]{1,3}|[0-9]{1,3})[\s-]?([A-Z]{1,3}|[0-9]{1,5}))$/.test(str),
+  'hu-HU': str => /^((((?!AAA)(([A-NPRSTVZWXY]{1})([A-PR-Z]{1})([A-HJ-NPR-Z]))|(A[ABC]I)|A[ABC]O|A[A-W]Q|BPI|BPO|UCO|UDO|XAO)-(?!000)\d{3})|(M\d{6})|((CK|DT|CD|HC|H[ABEFIKLMNPRSTVX]|MA|OT|R[A-Z]) \d{2}-\d{2})|(CD \d{3}-\d{3})|(C-(C|X) \d{4})|(X-(A|B|C) \d{4})|(([EPVZ]-\d{5}))|(S A[A-Z]{2} \d{2})|(SP \d{2}-\d{2}))$/.test(str),
+  'pt-BR': str =>
+    /^[A-Z]{3}[ -]?[0-9][A-Z][0-9]{2}|[A-Z]{3}[ -]?[0-9]{4}$/.test(str),
   'pt-PT': str =>
     /^([A-Z]{2}|[0-9]{2})[ -·]?([A-Z]{2}|[0-9]{2})[ -·]?([A-Z]{2}|[0-9]{2})$/.test(str),
   'sq-AL': str =>
     /^[A-Z]{2}[- ]?((\d{3}[- ]?(([A-Z]{2})|T))|(R[- ]?\d{3}))$/.test(str),
-  'pt-BR': str =>
-    /^[A-Z]{3}[ -]?[0-9][A-Z][0-9]{2}|[A-Z]{3}[ -]?[0-9]{4}$/.test(str),
   'sv-SE': str =>
     /^[A-HJ-PR-UW-Z]{3} ?[\d]{2}[A-HJ-PR-UW-Z1-9]$|(^[A-ZÅÄÖ ]{2,7}$)/.test(str.trim()),
-  'en-IN': str => /^[A-Z]{2}[ -]?[0-9]{1,2}(?:[ -]?[A-Z])(?:[ -]?[A-Z]*)?[ -]?[0-9]{4}$/.test(str),
 };
 
-validators['hi-IN'] = validators['en-IN'];
-validators['gu-IN'] = validators['en-IN'];
-validators['as-IN'] = validators['en-IN'];
-validators['bn-IN'] = validators['en-IN'];
-validators['kn-IN'] = validators['en-IN'];
-validators['ml-IN'] = validators['en-IN'];
-validators['mr-IN'] = validators['en-IN'];
-validators['or-IN'] = validators['en-IN'];
-validators['pa-IN'] = validators['en-IN'];
-validators['sa-IN'] = validators['en-IN'];
-validators['ta-IN'] = validators['en-IN'];
-validators['te-IN'] = validators['en-IN'];
-validators['kok-IN'] = validators['en-IN'];

```

Comment:
Why are we removing this (also from the README)?

Reply:
Was reverting something that I'd missed in past review. We didn't need specific region/language locales for license plates that are already Latin/EN based. Actually at some point, we were to suggest that we just use country-codes only for such validators.

Reply:
Makes sense, I have created https://github.com/validatorjs/validator.js/issues/2171 with some of my thoughts on it